File: src/main/kotlin/basic_control_structures/exercise_5.kt
package basic_control_structures

// Write a program that reads a student's two grades in a subject and displays their average on the screen.
fun averageGrade() {
    print("1st grade: ")
    val grade1 = readln().toDouble()

    print("2nd grade: ")
    val grade2 = readln().toDouble()

    val average = grade1.plus(grade2) / 2

    println("The average grade between $grade1 and $grade2 is $average.")
}

File: Dart/a_basic_control_structure/exercise_5.dart
// Write a program that reads a student's two grades in a subject and displays their average on the screen.

import 'dart:io';

void main() {
  stdout.write('1st grade: ');
  double firstValue = double.parse(stdin.readLineSync()!);

  stdout.write('2nd grade: ');
  double secondValue = double.parse(stdin.readLineSync()!);

  averageGrade(firstValue, secondValue);
}

void averageGrade(double value1, double value2) {
  double average = (value1 + value2) / 2;
  print('The average grade between $value1 and $value2 is $average.');
}
